### PR TITLE
Add title for head

### LINKED ISSUE #20: Add title for head(BiHjoGrafen)
Title of the document

### PR DESCRIPTION
**Pull request by MadeleineDahlqvist**
Added the title in all three HTML files. 

Fixes #20 (Links to the issue in projects)

### CODE REVIEW

**Review by AntonisTerzo — APPROVED** [review]
Geat job with the title Madde!

File: pages/aboutUs.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
 <head>
   <meta charset='utf-8'>
   <meta http-equiv='X-UA-Compatible' content='IE=edge'>
-  <title>title</title>
+  <title>BiHjoGrafen</title>
```
Comment (ANschoolacc) [inline pages/aboutUs.html:6]: Maybe make it more specific? Like "BiHjoGrafen - Om oss"

File: pages/movies.html [hunk 2]
```diff
@@ -4,7 +4,7 @@
 <head>
   <meta charset='utf-8'>
   <meta http-equiv='X-UA-Compatible' content='IE=edge'>
-  <title>title</title>
+  <title>BiHjoGrafen</title>
```
Comment (ANschoolacc) [inline pages/movies.html:7]: Maybe make it more specific? Like "BiHjoGrafen - Filmer"

**Review by ANschoolacc — COMMENTED** [review]
Maybe be more specific with the title names on aboutUs.html and movies.html.

Otherwise, looks good!

File: pages/aboutUs.html (hunk 1 above)
Comment (MadeleineDahlqvist) [reply pages/aboutUs.html:6]: Good point, I will fix that!

**Pull request merged**